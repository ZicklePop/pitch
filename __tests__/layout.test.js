import React from 'react'
import renderer from 'react-test-renderer'
import Layout from '../components/layout'

const MOCK = {
  title: 'Rae the Doe',
  description: 'A webcomic about a gay disaster, a punk skunk, and lots and lots of puns. Created by Olive Rae Brinker. Updates Monday, Wednesday and Friday.'
}

describe('component - layout', () => {
  it('should render', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('should render with props', () => {
    const component = renderer.create(<Layout {...MOCK}><b>#content</b></Layout>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
