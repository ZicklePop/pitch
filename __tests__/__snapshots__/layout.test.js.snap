// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component - layout should render 1`] = `
<main
  className="jsx-2152329932  "
/>
`;

exports[`component - layout should render with props 1`] = `
<main
  className="jsx-2152329932  "
>
  <b>
    #content
  </b>
</main>
`;
